src/components/map/toolbar/toolbarSubmenus.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ToolbarTools from '../../../enums/toolbarTools';
+
+const submenus = {
+    edit: (
+        <div>
+            Voit nyt raahata solmut ja pysäkit kartalla.
+        </div>
+    ),
+};
+
+const getSubmenu = (tool: ToolbarTools): JSX.Element | null => {
```

Comment:
I would make this as toolHelp.tsx react component with render function returning:
```
render() {
if(this.props.tool != ToolbarTool.Edit) return null;

return (
<div>
       Voit siirtää pysäkkejä tai risteyksiä raahaamalla niitä kartalla.
</div>
)}
```

Later if we decide to have some other things in this menu, we can make a wrapper and put things inside, for example:
```
<wrapper>
    <ToolHelp />
   <SnapMenu />
</wrapper>
```